scripts/qa_crowbarsetup.sh

```diff
@@ -1736,6 +1738,16 @@ function custom_configuration()
                 novanodes="[ ${novanodes%,} ]"
                 proposal_set_value nova default "['deployment']['nova']['elements']['nova-multi-compute-${libvirt_type}']" "$novanodes"
             fi
+
+            if [ -n "$want_sles12" ] && [ -n "$want_docker" ] ; then
+                proposal_set_value nova default "['deployment']['nova']['elements']['nova-multi-compute-docker']" "['$sle12node']"
+                # do not assign another compute role to this node
+                local novanodes
+                novanodes=`printf "%s\n" $novanodes | grep -iv $sle12node`
+                novanodes="[ ${novanodes} ]"
```

Comment:
This is still not a valid ruby value.
It mus look like this:
```
[ "value1", "value2", "value3" ]
```
Your code creates this:
```
[ value1  value2  value3 ]
```